docs/source/restore/compatibility-matrix.rst

```diff
@@ -11,6 +11,9 @@ The following table shows which version of Scylla Manager restore task supports
    * - ScyllaDB Manager Version
      - ScyllaDB Open Source Version
      - ScyllaDB Enterprise Version
+   * - 3.2
+     - 5.0, 5.1, 5.2
```

Comment:
Can we smoke test and add 5.4 here?
RC1 is available

Reply:
I will trigger the following tests for SM 3.2 and Scylla 5.4.0-rc1:
- [SM repo tests](https://github.com/scylladb/scylla-manager/actions/runs/6763490314/)
- [Dtests](https://jenkins.scylladb.com/view/scylla-manager/job/manager-master/job/manager-dtest/1346/)
- [centos-sanity-test](https://jenkins.scylladb.com/view/scylla-manager/job/manager-master/job/centos-sanity-test/) - queued

Reply:
Tests are failing, but it looks like a problem with config changes. I will investigate that.
E.g. with some changes to testing `scylla.yaml` in SM repo, SM tests pass when cluster is created successfully, but the process of building it sometimes fails.